crates/sui-core/src/authority.rs

```diff
@@ -1032,7 +1030,7 @@ impl AuthorityState {
     /// When Ok, returns None if the transaction has not been executed, and returns
     /// (TransactionEffects, TransactionEvents) if the transaction has been executed.
     #[instrument(level = "trace", skip_all)]
-    pub async fn handle_transaction_v2(
+    pub(crate) fn vote_transaction(
```

Comment:
nit: can you change the debug stmt and metric in the method to match the name change